api/common.js

```diff
@@ -413,6 +414,87 @@ exports.load_github_detail = function(service_name, cb) {
     }).catch(cb)
 }
 
+
+exports.generateQuery = function(str){
```

Comment:
You should avoid using `function`. Use the arrow syntax instead. Arrow syntax makes sure that `this` won't be bound to the caller index. I don't really use `this` on backend scripts, but this is important for front end where we use Vue's this pointer a lot. To keep our coding style consistent, I think we should avoid using `function` unless it's absolutely necessary.